tests/integration/CreateSprintTemplate.test.ts

```diff
@@ -0,0 +1,143 @@
+import { afterAll, beforeAll, describe, it } from "vitest";
+import { expect } from "chai";
+import puppeteer from "puppeteer";
+import type { Browser, Page } from "puppeteer";
+import dotenv from "dotenv";
+import { nanoid } from "nanoid";
+import { installMouseHelper } from "./install-mouse-helper.js";
+
+dotenv.config();
+const username = process.env.TEST_FIXTURE_USERNAME;
+const password = process.env.TEST_FIXTURE_PASSWORD;
+
+const randomTemplateName = nanoid(10);
+describe("Login Page:", () => {
+  let browser: Browser;
+  let page: Page;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      // headless: false, // <------------------------ This is the line that makes it so you can see whats going on
+
+      slowMo: 50, // <-------- use this if you want to slow down the tests so you can see what is happening, 50 minimum to make it work properly or its too fast.
+      defaultViewport: null,
+      args: ["--window-size=1400,1022"], // In headed mode you need the window size to have a width of at least 1400px or the drag test will fail as there isnt enough window space to drag the box to the correct location
+    });
+    page = await browser.newPage();
+    await installMouseHelper(page); // This is a helper function that allows you to see what the mouse is actually doing on the screen in headed mode.
+  });
+
+  afterAll(async () => {
+    await browser.close();
```

Comment:
Should probably log out before closing the browser?

Reply:
Ya logout is the final thing before it takes a screenshot if it fails, I can move it up there if you want to.